website/pages/component/side-nav/example.tsx

```diff
@@ -57,18 +58,33 @@ function DefaultExample({
       } as const);
 
   return (
-    <SideNav>
-      <SideNavGroup {...collapsibleProps} header={header}>
-        <SideNavItem active={active}>Projects</SideNavItem>
-        <SideNavItem disabled={disabled}>Alerts</SideNavItem>
-        <SideNavItem>Activity Feed</SideNavItem>
-        <SideNavItem>Settings</SideNavItem>
-        <SideNavItem>Access Manager</SideNavItem>
-        <SideNavItem>Billing</SideNavItem>
-        <SideNavItem>Support</SideNavItem>
-        <SideNavItem>{children}</SideNavItem>
-      </SideNavGroup>
-    </SideNav>
+    <div
+      className={css`
+        display: grid;
+        grid-template-columns: auto 1fr;
+        height: 500px;
+      `}
+    >
+      <SideNav>
+        <SideNavGroup {...collapsibleProps} header={header}>
```

Comment:
I think we should update the toggles to better show off what this side nav can do / achieve parity with storybook

Reply:
I updated the knobs a bit – let me know if there's something specific you'd like to see that's not there.